app/models/inventory.rb

```diff
@@ -9,15 +9,24 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #  owner_id    :integer
+#  message     :text
+#  assigned_at :datetime
 #
 
 class Inventory < ActiveRecord::Base
   include Authority::Abilities
+
+  default_scope { order(created_at: :desc) }
+
   has_many :product_entries
   has_many :data_entries
   has_many :access_requests, class_name: 'InventoryAccessRequest'
+  has_many :messages, foreign_key: :tool_id
```

Comment:
The `Assessment` model appends `dependent: :destroy` to this. Are we sure we don't want it here too?